src/NamedItemContainer.js

```diff
@@ -17,73 +17,53 @@ class NamedItemContaner {
   }
 
   async getNamedItems() {
-    try {
-      const client = await this._oneDrive.getClient();
-      const result = await client.get(`${this.uri}/names`);
-      return result.value.map((v) => ({
-        name: v.name,
-        value: v.value,
-        comment: v.comment,
-      }));
-    } catch (e) {
-      this.log.error(StatusCodeError.getActualError(e));
-      throw new StatusCodeError(e.message, e.statusCode || 500);
-    }
+    const result = await this._oneDrive.doFetch(`${this.uri}/names`);
+    return result.value.map((v) => ({
+      name: v.name,
+      value: v.value,
+      comment: v.comment,
+    }));
   }
 
   async getNamedItem(name) {
     try {
-      const client = await this._oneDrive.getClient(false);
-      return await client.get(`${this.uri}/names/${name}`);
+      return this._oneDrive.doFetch(`${this.uri}/names/${name}`);
     } catch (e) {
       if (e.statusCode === 404) {
         return null;
       }
-      this.log.error(StatusCodeError.getActualError(e));
```

Comment:
why was the logging removed?

Reply:
I think it is legitimate to remove that log statement because the error is rethrown.

Reply:
> I think it is legitimate to remove that log statement because the error is rethrown.

yes, the error is already logged in `doFetch()`